packages/nimble-components/src/table-column/mixins/groupable-column.ts

```diff
@@ -26,11 +26,7 @@ export function mixinGroupableColumnAPI<
         }
 
         public groupIndexChanged(): void {
-            if (typeof this.groupIndex === 'number') {
-                this.columnInternals.groupIndex = this.groupIndex;
-            } else {
-                this.columnInternals.groupIndex = undefined;
-            }
+            this.columnInternals.groupIndex = this.groupIndex ?? undefined;
```

Comment:
Can you do the same in the fractional-width-column mixins for fractionalWidthChanged and minPixelWidthChanged?